### PR TITLE
Fix delays of E4A

### PR DESCRIPTION
**Pull request by universmile**
We translate information delays into material delays, thus improving #141.

### CODE REVIEW

**Review by natema — COMMENTED** [review]
I tried to run the plots in `plots.jl`. 
`INEQUALITY_INDEX_1980`, `AVERAGE_WELLBEING_INDEX_1` and `ENERGY_USE_Mtoe_per_y` are undefined. 
I could plot `Social_tension_1` (*Social Tension Giant Leap*) which matches (:tada:) and `Wellbeing_effect_of_participation_1` (*Wellbeing Giant Leap*) which doesn't match (:disappointed:). 
Here are the screenshots: 
image
image

**Comment by natema** [comment]
I'm ok merging this PR since it fixes the DELAY issue on which we have been stuck for long, but if the missing and mismatching plots are not immediate to fix, before merging I'd ask @universmile to open corresponding issues.

**Review by natema — APPROVED** [review]

File: src/Earth4All/Earth4All.jl [hunk 1]
```diff
@@ -898,6 +888,17 @@ inits[:Owner_consumption_G_dollar_per_y] = inits[:Permanent_owner_cash_inflow_G_
...
+inits[:Deaths_Mp_per_y1] = p[:Dying_in_1980_Mp_per_y] * inits[:LE_at_60_y] / 10
+inits[:Deaths_Mp_per_y2] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y3] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y4] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y5] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y6] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y7] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y8] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y9] = inits[:Deaths_Mp_per_y1]
+inits[:Deaths_Mp_per_y10] = inits[:Deaths_Mp_per_y1]
```
Comment (aurorarossi) [inline src/Earth4All/Earth4All.jl:901]: Why here only var1 is multiplied by (delay/order) while for passing_20/40/60 it is done for all var?

Comment (universmile) [reply src/Earth4All/Earth4All.jl:901]: It is actually done for all variables. Notice that variables var2, ..., var10 are receiving the same initialisation as var1.

Comment (aurorarossi) [reply src/Earth4All/Earth4All.jl:901]: yes true, I didn't realize that.

**Pull request merged**